ansible_collections/arista/avd/plugins/filter/add_md_toc.py

```diff
@@ -18,6 +18,59 @@
 except ImportError:
     HAS_MD_TOC = False
 
+DOCUMENTATION = r"""
```

Comment:
Any reason to case Markdown to MarkDown?

Reply:
only because it was so already. Changing.